csp_billing_adapter/csp_cache.py

```diff
@@ -109,14 +110,23 @@ def cache_meter_record(
     :param next_bill_time:
         The time after which the next billing submission should be
         performed.
+    :param billed_records:
+        The list of records used to generate the most recent bill.
     """
+    cache = hook.get_cache(config=config)
```

Comment:
Is it possible to pass in an existing copy of cache or records here to save the extra load?

Reply:
Yeah we could potentially move the construction of the remaining_usage_records into process_metering() at the same time that we calculate the billed records

Reply:
Yeah, it will probably be teased out in the future if not now. For https://github.com/SUSE-Enceladus/csp-billing-adapter/issues/65